From 159df2b2f11adbc94f14eab0806202476941ed5e Mon Sep 17 00:00:00 2001
Message-ID: <159df2b2f11adbc94f14eab0806202476941ed5e.1732837202.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732837202.git.sharlatanus@gmail.com>
References: <cover.1732837202.git.sharlatanus@gmail.com>
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Sun, 24 Nov 2024 13:27:24 +0300
Subject: [PATCH 03/15] gnu: Add go-github-com-makenowjust-heredoc.

* gnu/packages/golang-xyz.scm (go-github-com-makenowjust-heredoc): New variable.

Change-Id: I686dfd6a798da764b004a53a562ca788e70f8cd5
Reviewed-by: Sharlatan Hellseher <sharlatanus@gmail.com>
---
 gnu/packages/golang-xyz.scm | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index 561b82af11..fefc5c6a97 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -4893,6 +4893,30 @@ (define-public go-github-com-logrusorgru-aurora-v4
     (native-inputs
      (list go-github-com-stretchr-testify))))
 
+(define-public go-github-com-makenowjust-heredoc
+  (package
+    (name "go-github-com-makenowjust-heredoc")
+    (version "1.0.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/makenowjust/heredoc")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "18f21zm8n2wlnkz1ylw8rcxmqxyv2rlz8749yfqggm2m0m2884pj"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:import-path "github.com/MakeNowJust/heredoc"))
+    (home-page "https://github.com/MakeNowJust/heredoc")
+    (synopsis "Here-documents with indent")
+    (description
+     "This package implements a functionality of creating here-documents from
+raw strings.")
+    (license license:expat)))
+
 (define-public go-github-com-marcinbor85-gohex
   ;; No release, see <https://github.com/marcinbor85/gohex/issues/5>.
   (let ((commit "baab2527a9a2a4abb3dc06baabedfa5e0268b8d8")
-- 
2.46.0

